cli/cmd/install.go

```diff
@@ -441,10 +441,10 @@ func (options *installOptions) buildValuesWithoutIdentity(configs *pb.All) (*ins
 		values.TapResources = &*defaultConstraints
 		values.WebResources = &*defaultConstraints
 
-		values.IdentityResources = &resources{
-			CPU:    constraints{Request: "10m"},
-			Memory: constraints{Request: "10Mi"},
-		}
+		// The identity controller maintains no internal state, so it need not request
+		// 50Mi.
+		values.IdentityResources = &*defaultConstraints
+		values.IdentityResources.Memory = constraints{Request: "10Mi"}
```

Comment:
Ugh, unfortunately, this modifies the memory request for all components using `defaultConstraints`. The issue is that `&*` isn't actually creating a new pointer -- it's copying the same pointer to all of the resource fields.

This fixes it, but it sure us ugly:

```diff
diff --git a/cli/cmd/install.go b/cli/cmd/install.go
index 384a4203..cb78b0e5 100644
--- a/cli/cmd/install.go
+++ b/cli/cmd/install.go
@@ -426,6 +426,15 @@ func (options *installOptions) buildValuesWithoutIdentity(configs *pb.All) (*ins
 			Proxy:   proxyJSON,
 			Install: installJSON,
 		},
+
+		DestinationResources:   &resources{},
+		GrafanaResources:       &resources{},
+		IdentityResources:      &resources{},
+		PrometheusResources:    &resources{},
+		ProxyInjectorResources: &resources{},
+		PublicAPIResources:     &resources{},
+		TapResources:           &resources{},
+		WebResources:           &resources{},
 	}
 
 	if options.highAvailability {
@@ -434,16 +443,16 @@ func (options *installOptions) buildValuesWithoutIdentity(configs *pb.All) (*ins
 			Memory: constraints{Request: "50Mi"},
 		}
 		// Copy constraints to each so that further modification isn't global.
-		values.DestinationResources = &*defaultConstraints
-		values.GrafanaResources = &*defaultConstraints
-		values.ProxyInjectorResources = &*defaultConstraints
-		values.PublicAPIResources = &*defaultConstraints
-		values.TapResources = &*defaultConstraints
-		values.WebResources = &*defaultConstraints
+		*values.DestinationResources = *defaultConstraints
+		*values.GrafanaResources = *defaultConstraints
+		*values.ProxyInjectorResources = *defaultConstraints
+		*values.PublicAPIResources = *defaultConstraints
+		*values.TapResources = *defaultConstraints
+		*values.WebResources = *defaultConstraints
 
 		// The identity controller maintains no internal state, so it need not request
 		// 50Mi.
-		values.IdentityResources = &*defaultConstraints
+		*values.IdentityResources = *defaultConstraints
 		values.IdentityResources.Memory = constraints{Request: "10Mi"}
 
 		values.PrometheusResources = &resources{
```

There's prob a better way to go about it.